src/platform/plugins/shared/home/public/application/components/solutions_section/__snapshots__/solution_panel.test.tsx.snap

```diff
@@ -1,25 +1,20 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`SolutionPanel renders the solution panel for the given solution 1`] = `
-<EuiFlexItem
```

Comment:
Another very weird case of enzyme snapshot test change... It just decided to remove this parent from from the snapshot :(